packages/mux-player/test/player.test.js

```diff
@@ -216,37 +188,75 @@ describe('<mux-player>', () => {
     assert.equal(muxVideo.preload, defaultPreload, `muxVideo default preload is ${defaultPreload}`);
   });
 
-  it('poster is forwarded to the media element', async function () {
+  it('poster is forwarded to the media-poster-image element', async function () {
     const player = await fixture(`<mux-player
       poster="https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0"
     ></mux-player>`);
-    const muxVideo = player.media;
+    const mediaPosterImage = player.theme.shadowRoot.querySelector('media-poster-image');
 
     assert.equal(
       player.poster,
-      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0'
+      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0',
+      'player.poster does not equal in-html defined poster attribute'
     );
     assert.equal(
-      muxVideo.poster,
-      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0'
+      mediaPosterImage.getAttribute('src'),
+      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0',
+      'mediaPosterImage src attribute does not equal in-html defined poster attribute'
     );
 
     player.removeAttribute('poster');
-    assert(!muxVideo.hasAttribute('poster'), `has poster attr removed`);
+    assert(!mediaPosterImage.hasAttribute('src'), `has src attr removed`);
 
     player.setAttribute(
       'poster',
       'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=1'
     );
     assert.equal(
-      muxVideo.getAttribute('poster'),
+      mediaPosterImage.getAttribute('src'),
       'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=1',
       `has poster attr added`
     );
+  });
+
+  it('poster can be unset with an empty string', async function () {
+    const player = await fixture(`<mux-player
+      poster="https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0"
+      playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
+      stream-type="on-demand"
+    ></mux-player>`);
+    const mediaPosterImage = player.theme.shadowRoot.querySelector('media-poster-image');
+
+    assert.equal(
+      player.poster,
+      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0',
+      'does not equal in-html defined poster attribute'
+    );
+
+    player.removeAttribute('poster');
     assert.equal(
-      muxVideo.poster,
+      player.poster,
+      'https://image.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/thumbnail.jpg',
+      'uses the derived poster if no poster attribute is present'
+    );
+
+    player.poster = '';
+    assert.equal(player.poster, '');
+    assert(!mediaPosterImage.hasAttribute('src'), 'media-poster-image does have a poster attribute');
```

Comment:
Should the comment be `does *not* have a poster`?

Reply:
yes I adapted this style from @cjpillsbury, the message is shown when the test fails so it makes sense to say what happened but yes it might be all mixed up now because I didn't used to write it that way with other tests 🤷

Reply:
yeah the "does" may be the confusing part. For a light lift on rephrasing standardization, we could phrase these as "should"/"shouldn't" statements directly related to the assertion. e.g. for this case:

> media-poster-image shouldn't have a `src` attribute set.